source/getting-started/index.md

```diff
@@ -35,7 +35,8 @@ If you get a *"command not found"* error or an outdated version for Node:
 
 * Windows or Mac users can download and run [this Node.js installer](http://nodejs.org/en/download/).
 * Mac users often prefer to install Node using [Homebrew](http://brew.sh/). After
-installing Homebrew, run `brew install node` to install Node.js.
+installing Homebrew, run `brew install node` to install Node.js. Alternatively, installer packages are available directly
+from [Node.js](https://nodejs.org/en/download/).
```

Comment:
Lately I've been recommending to new developers to install it this way. For some reason I've helped a half-dozen people fix where they've somehow installed it with `sudo` and then can't install global npm packages without elevated privileges. The installers mitigate that.